src/cpu/cpu_convolution_pd.hpp

```diff
@@ -58,10 +53,45 @@ struct cpu_convolution_fwd_pd_t : public convolution_fwd_pd_t {
         }
         return !is_zero_preserved;
     }
+
+protected:
+    // See `convolution_pd_t::attr_scales_ok` comment.
+    status_t attr_scales_ok(
+            const std::unordered_map<int, std::vector<int>> &supported_args_map
+            = {{DNNL_ARG_SRC, {0}}, {DNNL_ARG_WEIGHTS, {0, 1}},
+                    {DNNL_ARG_DST, {0}}}) const {
+        return this->convolution_fwd_pd_t::attr_scales_ok(supported_args_map);
+    }
+
+    // See `convolution_pd_t::attr_zero_points_ok` comment.
+    // Put a default map relevant for most implementations. Once it's changed,
+    // update the default.
+    status_t attr_zero_points_ok(
+            const std::unordered_map<int, std::vector<int>> &supported_args_map
+            = {{DNNL_ARG_SRC, {0}}, {DNNL_ARG_DST, {0}}}) const {
+        return this->convolution_fwd_pd_t::attr_zero_points_ok(
```

Comment:
(nit) It's not necessary to use `this` here. You could just call `convolution_fwd_pd_t::attr_zero_points_ok(...)`.

Reply:
It's a remaining product of a previous solution. Removed, thanks.